ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -135,4 +135,170 @@ spec:
     SELECT 1 FROM security_profile_info WHERE clear_text_password = 0;
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS-win10-stand-alone-1.1.7
-  contributors: marcosd4h
\ No newline at end of file
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Digitally encrypt or sign secure channel data (always)' is set to 'Enabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting determines whether all secure channel traffic that is initiated by the
+    domain member must be signed or encrypted.
+    When a computer joins a domain, a computer account is created. After it joins the domain, the
+    computer uses the password for that account to create a secure channel with the Domain
+    Controller for its domain every time that it restarts. Requests that are sent on the secure
+    channel are authenticated—and sensitive information such as passwords are encrypted—but the
+    channel is not integrity-checked, and not all information is encrypted.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via GP, set the
+    following UI path to 'Enabled':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\Local Policies\Security Options\Domain member: Digitally encrypt or sign secure channel data (always)'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\Netlogon\Parameters\RequireSignOrSeal' AND data != 0);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS-win10-enterprise-2.3.6.1
```

Comment:
We agreed to have this format: `Tag for doc, tag for bullet`
In this case:
`tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_2.3.6.1`

Reply:
Applies for all items of course

Reply:
Should be fixed, if I understood correctly